src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/step/readwrite/SetReadWriteStep.kt

```diff
@@ -58,18 +60,17 @@ class SetReadWriteStep(
             failed = true
             val mutableInfo = mutableMapOf("message" to "Failed to set index to read-write")
             val errorMessage = e.message
-            if (errorMessage != null) mutableInfo.put("cause", errorMessage)
+            if (errorMessage != null) mutableInfo["cause"] = errorMessage
             info = mutableInfo.toMap()
         }
     }
 
     override fun getUpdatedManagedIndexMetaData(currentMetaData: ManagedIndexMetaData): ManagedIndexMetaData {
         return currentMetaData.copy(
-            step = name,
-            stepStartTime = getStepStartTime().toEpochMilli(),
-            transitionTo = null,
```

Comment:
Also here (if we don't need to reset this anymore then these are all just spam now 😄 )